packages/contracts-bedrock/src/cannon/PreimageOracle.sol

```diff
@@ -183,6 +183,90 @@ contract PreimageOracle is IPreimageOracle {
         preimageLengths[key] = size;
     }
 
+    /// @inheritdoc IPreimageOracle
+    function loadBlobPreimagePart(
```

Comment:
Can we add unit tests for this function?

Reply:
Unfortunately not yet since there's no Cancun support in foundry :(